backend/src/controllers/coordinatorController.js

```
@@ -47,6 +47,18 @@ const approveOrganization = async (req, res) => {
   }
 };
 
+
+const getAllOrganizations = async (req, res) => {
+  try {
+    const { page = 1, limit = 10 } = req.query;
+    const offset = (parseInt(page) - 1) * parseInt(limit);
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)